pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Enterprise apps on Amazon EKS. In about a day.
+excerpt: Build battle-tested Amazon EKS clusters, designed for enterprise with security, observability, and usability built-in. Manage 100% of your infrastructure using code, all in ~1 day. <b>Join now for special early access pricing!</b>
+permalink: /eks-for-enterprise/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+technologies:
+- name: Spring Boot
+  logo: /assets/img/logos-technologies/logo-spring-boot.png
+- name: Ruby on Rails
+  logo: /assets/img/logos-technologies/logo-rails.png
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
+        <pre>
+            <code class="language-yml">
+            # Configure CIS Benchmark compliance
+            cis:
+              enabled: true
```

Comment:
Would we ever want someone to disable this?

Reply:
Perhaps instead, we should have settings here...

E.g.,

```yaml
cis:
  enable_pod_security_policies: true
  enable_network_security_policies: true
  ... etc
```

Reply:
The other advantage of this is that many of our enterprise customers have asked about pod and network security policies for K8S, so calling that out explicitly here may resonate.

Reply:
Makes sense. Added this in.